flair/datasets/sequence_labeling.py

```diff
@@ -447,6 +447,53 @@ def __init__(
             document_separator_token=None if not document_as_sequence else "-DOCSTART-",
         )
 
+class MIT_Restaurants(ColumnCorpus):
+    def __init__(
+            self,
+            base_path: Union[str, Path] = None,
+            tag_to_bioes: str = "ner",
+            in_memory: bool = True,
+            document_as_sequence: bool = False,
+    ):
+        """
+        Initialize the experimental MIT Restaurant corpus available on https://groups.csail.mit.edu/sls/downloads/restaurant/.
+        The first time you call this constructor it will automatically download the dataset.
+        :param base_path: Default is None, meaning that corpus gets auto-downloaded and loaded. You can override this
+        to point to a different folder but typically this should not be necessary.
+        :param tag_to_bioes: NER by default, need not be changed, but you could also select 'pos' to predict
+        POS tags instead
+        :param in_memory: If True, keeps dataset in memory giving speedups in training.
+        :param document_as_sequence: If True, all sentences of a document are read into a single Sentence object
+        """
+        if type(base_path) == str:
+            base_path: Path = Path(base_path)
+
+        # column format
+        columns = {0: "text", 1: "ner"}
+
+        # this dataset name
+        dataset_name = self.__class__.__name__.lower()
+
+        # default dataset folder is the cache root
+        if not base_path:
+            base_path = Path(flair.cache_root) / "datasets"
+        data_folder = base_path / dataset_name
+
+        # download data if necessary
+        mit_restaurants_path = "https://megantosh.s3.eu-central-1.amazonaws.com/MITRestoCorpus/"
+        cached_path(f"{mit_restaurants_path}dev.txt", Path("datasets") / dataset_name)
```

Comment:
There is no `dev.txt` in the MIT restaurants dataset. This downloads an empty file.

Reply:
Ah great! I was not aware Flair does  the sampling. Here is the new
Corpus size: 
 Training split: 6894 
 Test split: 1521 
 Dev split:766
as they show locally and from the downloaded corpus